.github/workflows/pipelines.yml

```
@@ -26,7 +26,7 @@ jobs:
     strategy:
       matrix:
         configuration: [Debug, Release]
-        framework: [net8.0,net6.0]
+        framework: [net9.0,net8.0]
```

yamllint

[warning] too few spaces after comma

(commas)